src/test_storageiface.py

```diff
@@ -0,0 +1,139 @@
+'''
+test_storageiface.py
+
+Basic unit testing of the storage interfaces
+'''
+
+import unittest
+import logging
+import configparser
+
+class TestStorage(unittest.TestCase):
+  '''Simple tests for the storage layers of the WOPI server'''
```

Comment:
I think we should clarify that one must configure the tests to run for each storage provider (or add tests for each provider separately). I'd assume that all providers were tested at once if I run this.